requirements/edx/development.txt

```diff
@@ -46,6 +46,7 @@ git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
 -e git+https://github.com/raccoongang/rg_instructor_analytics_log_collector@analytics2/releases/1.2.X#egg=rg_instructor_analytics_log_collector
 -e git+https://github.com/raccoongang/django-oauth2-provider@1.2.5-multiple-redirect_uris#egg=edx-django-oauth2-provider==1.2.5
 -e git+https://github.com/raccoongang/edx-search@1.2.1-rg#egg=edx-search==1.2.1-rg
+-e git+git@gitlab.raccoongang.com:ferret/microsite-reports.git@v0.0.2#egg=microsite_reports
```

Comment:
Won't we have issues with deployment here, since it's a private repo?
@a-kryachko can you please take a look?

Reply:
its okay.
`requirements/edx/development.txt` deployed only on `devstack` environment.
to deploy to fullstack envs `EDXAPP_EXTRA_REQUIREMENTS` is used and configured [here](https://gitlab.raccoongang.com/DevOps/awx-deployment/-/blob/master/inventories/edx/new-university/group_vars/prod/edx.yml#L216)

Reply:
fixed, thanks